tests/app/log/test_plugin.py

```diff
@@ -273,3 +274,17 @@ def test_start_logging2(self, app_dir):
             assert not isinstance(sys.stderr, StreamToLogRedirector)
         finally:
             sys.stdout = old
+
+    def test_display_rotating(self, app_dir, exopy_qtbot):
+        """Test the log display window
+
+        """
+        cmd_args = CMDArgs()
+        cmd_args.nocapture = True
```

Comment:
This line is not needed I believe.

Reply:
cmd_args.no_capture needs to be set because it is tested in exopy/app/log/manifest.enaml.
True seems to be the logical default value.